specification/indices/_types/IndexSettings.ts

```diff
@@ -420,7 +420,8 @@ export class MappingLimitSettings {
   nested_objects?: MappingLimitSettingsNestedObjects
   field_name_length?: MappingLimitSettingsFieldNameLength
   dimension_fields?: MappingLimitSettingsDimensionFields
-  ignore_malformed?: boolean
+  source?: MappingLimitSettingsSourceFields
+  ignore_malformed?: boolean | string
```

Comment:
I accidentally added this other change. Like all settings, it can be a string, which happens in the `cluster.get_component_templates` response recordings we have.